src/io/flutter/run/bazelTest/FlutterBazelTestConfigurationEditorForm.form

```diff
@@ -1,94 +1,119 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you attach a screen shot for this UI?

Reply:
I shared a screenshot for this internally.